var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,97 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    # make -j is currently broken
+    parallel = False
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
+    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
+
+    variant('shared', default=True,
+            description='Builds a shared version of the library')
 
-    version('1.9.0', '310ff02388bbaa2b1c1710ee970b5678')
+    variant('fortran', default=True,
+            description='Enable Fortran bindings support')
+
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('infiniband', default=False, description='Enable infiniband support')
```

Comment:
@pnorbert is that `--with-infiniband` option necessary to be set manually? I just copied it from the previous code but never used it before (gosh, maybe I should have on our local cluster...?)